clicache/src/impl/PdxLocalWriter.hpp

```diff
@@ -263,7 +270,7 @@ namespace Apache
           /// </summary>
           /// <param name="fieldName">The name of the field associated with the value.</param>
           /// <param name="ushortArray">The ushortArray to write.</param>
-          virtual IPdxWriter^ WriteUnsignedShortArray(String^ fieldName, array<System::UInt16>^ ushortArray);
+          virtual IPdxWriter^ WriteUShortArray(String^ fieldName, array<System::UInt16>^ ushortArray);
```

Comment:
Why shorten the names of these?  This seems unnecessary....

Reply:
Discussed above in an earlier conversation. Short, Int, and Long are the C# keywords used in Microsoft docs.